tests/cephfs/cephfs_bugs/test_rm_on_lost_found_dir.py

```diff
@@ -0,0 +1,133 @@
+            cmd=f"cd {fuse_mount_dir};"
+            f"mkdir subdir;"
+            f"dd if=/dev/urandom of=subdir/sixmegs bs=1M conv=fdatasync count=6 seek=0;",
+        )
+
+        out, rc = client1.exec_command(
+            sudo=True, cmd=f"rados ls -p cephfs.{fs_name}.data -f json"
+        )
+        rados_obj_ls = json.loads(out)
+        log.info("Delete one of the rados object")
+        if not rados_obj_ls:
+            raise CommandFailed(
+                "Can not execute the test cases as there are no rados objects created"
+            )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"rados rm {rados_obj_ls[0]['name']} -p cephfs.{fs_name}.data",
+        )
+        client1.exec_command(
+            sudo=True, cmd=f"cephfs-table-tool {fs_name}:0 reset session"
+        )
+        log.info("Fail File System")
+        client1.exec_command(sudo=True, cmd=f"ceph fs fail {fs_name}")
+        log.info("Reset sessions snaps and inodes")
+        cmd_list = [
+            f"cephfs-table-tool {fs_name}:0 reset session",
+            f"cephfs-table-tool {fs_name}:0 reset snap",
+            f"cephfs-table-tool {fs_name}:0 reset inode",
+        ]
+        for cmd in cmd_list:
+            client1.exec_command(sudo=True, cmd=cmd)
+
+        log.info("Reset the filesystem")
+        client1.exec_command(
+            sudo=True, cmd=f"ceph fs reset {fs_name} --yes-i-really-mean-it"
+        )
+        log.info("Reset journal")
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cephfs-journal-tool --rank={fs_name}:0 journal reset --force",
+        )
+
+        log.info("Initiate recovery tools on filesystem")
+        cmd_list = [
+            f"cephfs-data-scan init --force-init --filesystem {fs_name}",
+            f"cephfs-data-scan scan_extents --filesystem {fs_name}",
+            f"cephfs-data-scan scan_inodes --filesystem {fs_name}",
+            f"cephfs-data-scan scan_links --filesystem {fs_name}",
+            f"cephfs-data-scan cleanup --filesystem {fs_name}",
+        ]
+        for cmd in cmd_list:
+            client1.exec_command(sudo=True, cmd=cmd)
+
+        log.info("Mark MDS as repaired")
+        client1.exec_command(sudo=True, cmd=f"ceph mds repaired {fs_name}:0")
+
+        out, rc = client1.exec_command(sudo=True, cmd=f"ls -lrt {fuse_mount_dir}")
+        if "lost+found" not in out:
+            raise CommandFailed("Unable to create lost+found directory")
+
+        client1.exec_command(sudo=True, cmd=f"rm -rf {fuse_mount_dir}/lost+found/*")
+        log.info("We are able to delete contents of lost+found dir")
+        return 0
+
+    except CommandFailed as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        client1.exec_command(sudo=True, cmd=f"umount {fuse_mount_dir}", check_ec=False)
```

Comment:
@AmarnatReddy - Can you also include the deletion of the mounting directory on client?